src/app/src/Map/Map.tsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+
+
```

Comment:
We can also get ride of these imports and empty lines since they are unused in this component.